verifier/verifier_test.go

```diff
@@ -310,6 +316,91 @@ func assertNotationVerification(t *testing.T, scheme signature.SigningScheme) {
 	}
 }
 
+func createMockOutcome(certChain []*x509.Certificate) *notation.VerificationOutcome {
+	return &notation.VerificationOutcome{
+		EnvelopeContent: &signature.EnvelopeContent{
+			SignerInfo: signature.SignerInfo{
+				SignedAttributes: signature.SignedAttributes{
+					SigningTime: time.Now(),
+				},
+				CertificateChain: certChain,
+			},
+		},
+		VerificationLevel: &trustpolicy.VerificationLevel{
+			Enforcement: map[trustpolicy.ValidationType]trustpolicy.ValidationAction{trustpolicy.TypeRevocation: trustpolicy.ActionEnforce},
+		},
+	}
+}
+
+func TestVerifyRevocation(t *testing.T) {
```

Comment:
1. Can you please add a test where certificate is revoked but trust policy is configured to log revocation. validate that verification succeeds but proper error message is shown to user 
2. Can you please add a test where certificate is revoked but trust policy is configured to skip revocation. Validate that no revocation calls are made.